### ISSUE TITLE
Incorrect Item.File info returned by Connect Server

### ISSUE THREAD

**Issue by snarlysodboxer** [1Password/connect#42]
Testing against version 1.5.5 of connect server, using v1.5.0 of connect-sdk-go. It's returning an empty File.Name, and an incorrect file ID in the ContentPath, but only when there's more than one file in the item, and only for that second file.

Do a GetItem, then loop over the Item.Files and print them. You'll see the first one is correct, and the others are not.

Correct first file:
```golang
&onepassword.File{
    ID:"wtnh5e6lmncudctdredacted",
    Name:"tls.crt",
    Section:(*onepassword.ItemSection)(nil),
    Size:1903,
    ContentPath:"/v1/vaults/ch3poqtcarbpviredacted/items/ygq6tyfo3adhzkredacted/files/wtnh5e6lmncudctdredacted/content",
    content:[]uint8(nil)
}
```

Incorrect second file: (notice how the ID does not match the one in ContentPath, and Name is empty when it should be "tls.key".)
```golang
&onepassword.File{
    ID:"tjpabagntylpvredacted",  // does not match ID in ContentPath
    Name:"",  // missing Name
    Section:(*onepassword.ItemSection)(0xc00171d580),
    Size:1675,
    ContentPath:"/v1/vaults/ch3poqtcarbpviredacted/items/ygq6tyfo3adhzkredacted/files/kgt4gkxbavhhbndvredacted/content",
    content:[]uint8(nil)
}
```

**Comment by codertyler** [comment]
Hi @snarlysodboxer, 

Thank you for raising this issue. We did an investigation and found the bug with the current version. We are already on it! Thank you again!

**Comment by snarlysodboxer** [comment]
@codertyler It looks like this was fixed 5 days ago in Connect Server 1.5.6? I tested and it's fixed. I think we can close this unless you were waiting to update for another reason.
